.azure/pipelines/ci.yml

```diff
@@ -22,6 +22,12 @@ variables:
   value: true
 - name: _TeamName
   value:  AspNetCore
+- name: _PublishUsingPipelines
+  value: true
+- name: _DotNetArtifactsCategory
+  value: .NETCORE
+- name: _DotNetValidationArtifactsCategory
+  value: .NETCOREVALIDATION # Reviwers: Should this be .NETCORE ?
```

Comment:
???

Reply:
I think this should match `_DotNetArtifactsCategory` - @riarenas can you confirm?

Reply:
While you're testing the changes, it should match so that you see everything going to the feed you would expect it to go for the real thing. Once you're satsified with your testing, the variable should be removed.

Reply:
1. Will change the value.
2. I left the `_DotNetValidationArtifactsCategory` setting in the Blazor repo. Does that matter?

Reply:
Will change the value.
> I left the _DotNetValidationArtifactsCategory setting in the Blazor repo. Does that matter?
Only if you plan to publish to the tools validation channel for your branch. The publishing for that channel will go to a different feed, but I don't believe you want to publish to that channel anyways.

Reply:
To make sure I have this right: As long as it's only my deleted branch that has a default channel mapping to the validation channel, it just doesn't matter?

Reply:
Correct